scripts/verdaccio.sh

```diff
@@ -57,34 +58,37 @@ then
 fi
 
 # Bump all package versions (allow publish from current branch but don't push tags or commit)
-yarn lerna version minor --force-publish --allow-branch `git branch --show-current` --no-push --yes
-commit_to_revert="HEAD~1"
-
-if [ "$ci" = true ];
-then
-# Get rid of npmrc file generated by install since it will block lerna publish
-  git checkout -- .
-fi
-
-# Publish packages to verdaccio
-yarn lerna publish from-package --registry $registry --yes
-
+yarn workspaces foreach --all --no-private version minor --deferred
+yarn version apply --all
+# set the npm registry because that will set it at a higher level, making local testing easier
 npm set registry $registry
+commit_to_revert="HEAD~0"
+
+# Publish packages anonymously to verdaccio
+yarn workspaces foreach --all --no-private -pt npm publish
 
 if [ "$ci" = true ];
 then
+  echo 'build prod docs'
+  cd starters/docs
+  yarn config set npmRegistryServer $registry
+  cd ../tailwind
+  yarn config set npmRegistryServer $registry
+  cd ../..
   # build prod docs with a public url of /reactspectrum/COMMIT_HASH_BEFORE_PUBLISH/verdaccio/docs
-  PUBLIC_URL=/reactspectrum/`git rev-parse HEAD~1`/verdaccio/docs make website-production
+  PUBLIC_URL=/reactspectrum/`git rev-parse HEAD~0`/verdaccio/docs make website-production
```

Comment:
this is the simplification based on lerna used to create an extra commit with tags, now we don't do that, so we can just use the current commit